lua/wikis/commons/Links.lua

```diff
@@ -51,6 +51,10 @@ local PREFIXES = {
 		stream = 'https://live.bilibili.com/',
 	},
 	['bilibili-stream'] = {'https://live.bilibili.com/'},
+	blasttv = {
+		'https://blast.tv/',
+		match = 'https://blast.tv/',
```

Comment:
Does different page type on blast.tv have diferent fixes? like blast.tv/players/.... and blast.tv/match/xxxx?